### ISSUE TITLE
Error - Config flow could not be loaded: {"message":"Invalid handler specified"}

### ISSUE THREAD

**Issue by mesut3000** [al-one/hass-xiaomi-miot#1917]
### Device model / 设备型号

Xiaomi Miot Auto

### HA core version / HA版本

2024.11.0b2

### Integrated mode / 集成方式

Cloud (云端模式)

### The problem / 问题详情

Can not add any integration, I get:

Error
Config flow could not be loaded: {"message":"Invalid handler specified"}

Removed and added integration again but didn't change.

### Entity attributes / 实体属性

```yaml
Can not add any integration, I get:

Error
Config flow could not be loaded: {"message":"Invalid handler specified"}

Removed and added integration again but didn't change.
```

**Comment by moti013** [comment]
Same here

**Comment by dreamsun007** [comment]
same

**Comment by al-one** [comment]
Fixed in v0.7.22
